tfexec/terraform_test.go

```diff
@@ -115,6 +115,11 @@ resource "aws_security_group" "bar" {}
 		t.Fatalf("expect to have changes")
 	}
 
+	// create local workspace folder
+	// this is a prerequisite before calling OverrideBackendToLocal
+	path := filepath.Join(terraformCLI.Dir(), "terraform.tfstate.d", workspace)
+	os.MkdirAll(path, os.ModePerm)
+
```

Comment:
I think the preparation has been no longer required because the dir will be created in the OverrideBackendToLocal.

Reply:
thanks for pointing that out !